app/assets/stylesheets/mailer/digest.css.scss

```diff
@@ -0,0 +1,94 @@
+$link-color: hsl(220,65%,40%);
+$base-font: "Helvetica Neue", Helvetica, Arial, sans-serif;
+$heading-font: "Gill Sans", Lato, sans-serif;
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+a {
+  color: $link-color;
+  text-decoration: none;
+}
+
+table.background {
+  width: 100%;
+}
+
+table.container {
+  width: 640px;
+  font-family: $base-font;
+  margin: 0 auto;
+
+  tr.header-top td {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3